<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>livraisonDialog</class>
 <widget class="QDialog" name="livraisonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color:#00BFFF	;
	border:2px solid black;
	border-radius:10px;
}

QPushButton:hover{
	transform:  scale(0.7);
	background-color:#ADD8E6	;
}
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: #fff;
	color: red;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #454544;
	font-weight: bold;
	font-size: 13px;

}


/*-----QPushButton-----*/
QPushButton
{
	
	background-color: #ff9c2b;
color: #000000;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{	
	background-color: #ff9c2b;

}


QPushButton::pressed
{
	background-color: #3d4ef2;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #5c55e9;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 5px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: #5c55e9;
    border: 1px solid #5c55e9;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #5c55e9;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #c2c7d5;
	color: #2a547f;
	border: none;
	padding: 5px;

}


/*-----QListView-----*/
QListView
{
	background-color: #5c55e9;
	color: #fff;
	font-size: 14px;
	font-weight: bold;
	show-decoration-selected: 0;
	border-radius: 4px;
	padding-left: -15px;
	padding-right: -15px;
	padding-top: 5px;

} 


QListView:disabled 
{
	background-color: #5c5c5c;

}


QListView::item
{
	background-color: #454e5e;
	border: none;
	padding: 10px;
	border-radius: 0px;
	padding-left : 10px;
	height: 32px;

}


QListView::item:selected
{
	color: #000;
	background-color: #fff;

}


QListView::item:!selected
{
	color:white;
	background-color: transparent;
	border: none;
	padding-left : 10px;

}


QListView::item:!selected:hover
{
	color: #fff;
	background-color: #5564f2;
	border: none;
	padding-left : 10px;

}


/*-----QTreeView-----*/
QTreeView 
{
	background-color: #fff;
	show-decoration-selected: 0;
	color: #454544;

}


QTreeView:disabled
{
	background-color: #242526;
	show-decoration-selected: 0;

}


QTreeView::item 
{
	border-top-color: transparent;
	border-bottom-color: transparent;

}


QTreeView::item:hover 
{
	background-color: #bcbdbb;
	color: #000;

}


QTreeView::item:selected 
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:active
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:disabled
{
	background-color: #525251;
	color: #656565;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #fff;
	border: 1px solid gray;
    color: #454544;
    gridline-color: gray;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #bcbdbb;
    color: #000;

}


QTableView::item:selected 
{
	background-color: #5c55e9;
    color: #fff;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
	background-color: #ced5e3;
	border: none;
    color: #fff;

}


QHeaderView::section
{
	color: #2a547f;
	border: 0px;
	background-color: #ced5e3;
	padding: 5px;

}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #5c55e9;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: #d8dce6;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: #d8dce6;

}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>561</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Brush Script MT</family>
       <pointsize>28</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Brush Script MT'; font-size:28pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:normal; color:#f0e68c; vertical-align:super;&quot;&gt;Express &lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:normal; color:#f0e68c; vertical-align:sub;&quot;&gt;Delivery&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>791</width>
       <height>461</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QTableView" name="tab_livraison">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>631</width>
         <height>501</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimerbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>110</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer Livraison</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Mapbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>160</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Map des Livraisons</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editerbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>60</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Editer Livraison</string>
       </property>
      </widget>
      <widget class="QPushButton" name="statbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>320</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitter">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>380</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouterbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>10</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Livraison</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tribtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>220</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier les Livraisons</string>
       </property>
      </widget>
      <widget class="QPushButton" name="paymentbtn">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>270</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Payment</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QTableView" name="tab_livraison_rech">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>621</width>
         <height>501</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="adresserech">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chercherbtn">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Chercher Livraison</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="stat">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>631</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>431</height>
        </rect>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="chat">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>751</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QPushButton" name="CHAT">
          <property name="text">
           <string>CHAT</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="tri">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>610</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>tri</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="insertbox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>661</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Please Insert Information</string>
     </property>
     <widget class="QLineEdit" name="cin">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom_Livreuer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel_Livreur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Diplome</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomLivreur">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="telLivreur">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="diplome">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>390</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addbtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>490</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quantite">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>450</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantite</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="editbox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>681</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Please Edit Information</string>
     </property>
     <widget class="QLineEdit" name="cinEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom_Livreuer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel_Livreur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Diplome</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomLivreurEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresseEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="telLivreurEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="diplomeEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>390</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>490</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantite </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quantiteedit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="cinSupp">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>150</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="suppBtn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>390</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="suppbox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
